ddp-workspace/projects/ddp-sdk/src/lib/components/activityForm/picklist/autocompleteActivityPicklistQuestion.component.ts

```diff
@@ -74,9 +80,15 @@ export class AutocompleteActivityPicklistQuestion extends BaseActivityPicklistQu
             takeUntil(this.ngUnsubscribe)
         );
 
-        const sortedPicklistGroups = this.sortPolicy.sortPicklistGroups(this.block.picklistGroups);
-        const sortedPicklistOptions = this.sortPolicy.sortPicklistOptions(this.block.picklistOptions);
-
+        let sortedPicklistGroups;
+        let sortedPicklistOptions;
+        if (this.shouldBeSorted) {
+            sortedPicklistGroups = this.sortPolicy.sortPicklistGroups(this.block.picklistGroups);
+            sortedPicklistOptions = this.sortPolicy.sortPicklistOptions(this.block.picklistOptions);
+        } else {
+            sortedPicklistGroups = this.block.picklistGroups;
+            sortedPicklistOptions = this.block.picklistOptions;
+        }
```

Comment:
Decision to sort or not to sort is encoded in SortPolicy. You just have to configure it depending on what you want for the question.
https://github.com/broadinstitute/ddp-angular/blob/develop/ddp-workspace/projects/ddp-sdk/src/lib/services/sortOrder.ts
The default is NONE
https://github.com/broadinstitute/ddp-angular/blob/69f7d745da2e89fec6a6b7e6a996f5c662cf6838/ddp-workspace/projects/ddp-sdk/src/lib/services/picklistSortingPolicy.service.ts#L8

for body locations we want ALPHABETICAL. For the other ones NONE.